src/renderer/assets/styles/animation.css

```
@@ -89,6 +89,175 @@
+    transform: translate3d(0, 0, 0);
+    opacity: 1;
+  }
+}
+
+.animation-chat-message-enter-assistant {
+  transform-origin: left top;
+  animation: animation-chat-message-enter-assistant 200ms cubic-bezier(0.16, 1, 0.3, 1) both;
+}
+
+.cs-mark-fill {
+  fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)